python/test/unit/language/test_core.py

```diff
@@ -2293,6 +2293,11 @@ def test_scan_layouts(M, N, src_layout, axis, device):
               instr_shape=[16, 8]),
     MmaLayout(version=(3, 0), warps_per_cta=[4, 1], ctas_per_cga=[1, 1], cta_split_num=[1, 1], cta_order=[1, 0],
               instr_shape=[16, 16, 16]),
+] if not is_hip() else [
+    BlockedLayout([1, 4], [8, 8], [4, 1], [1, 0], [1, 1], [1, 1], [0, 1]),
```

Comment:
is there a better way to generalize for targets with different number of threads ?

Reply:
yeah. You can use something like `[4, THREADS_PER_WARP // 4]`

Reply:
can we unify the blocked_layout ones with the number of threads per warp var?

Reply:
Ah I see, didn't notice that variable. Sounds good!